### PR TITLE
CharmEditor Embed improvements

### PR DESCRIPTION
**Pull request by mattcasey**
- smoother resizing of sizes
- allow users to resize width of iframe
- smarter parsing for width/height attributes from pasted code
- add an "Embed" and "Link" tabs when inputting a iframe src, however we wanted to support embed codes no matter which tab you use - this is just a trick to educate users that they can use embed codes

### CODE REVIEW

File: components/common/CharmEditor/CharmEditor.tsx [hunk 1]
```diff
@@ -489,15 +489,6 @@ function CharmEditor({
       color: 'var(--charmeditor-active)'
     }
   });
-
-  const onResizeStop = useCallback(
```
Comment (mattcasey) [inline components/common/CharmEditor/CharmEditor.tsx:493]: this turned out not to be necessary

File: components/common/CharmEditor/specs/embeddedNodeSpec.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export function embeddedNodeSpec(_spec: EmbeddedSpecProps): RawSpecs {
     name: _spec.name,
     markdown: _spec.markdown,
     schema: {
+      atom: true,
```
Comment (mattcasey) [inline components/common/CharmEditor/specs/embeddedNodeSpec.ts:13]: I think this is what really made resizing not go haywire. Atom tells Prosemirror that the content inside is not editable

File: components/common/CharmEditor/components/Resizable/Resizable.tsx [hunk 3]
```diff
@@ -40,6 +35,10 @@ function Resizable(props: ResizableProps) {
     }
   }, []);
 
+  useEffect(() => {
+    setSize(initialSize);
+  }, [initialSize]);
```
Comment (mattcasey) [inline components/common/CharmEditor/components/Resizable/Resizable.tsx:40]: this also seemed to help smooth out resizing

File: components/common/CharmEditor/components/video/video.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ export function plugins() {
   return [
     NodeView.createPlugin({
       name: 'video',
-      containerDOM: ['video-embed']
+      containerDOM: ['div', { class: 'video-container' }]
```
Comment (mattcasey) [inline components/common/CharmEditor/components/video/video.ts:11]: using a custom tag "video-embed" was causing some CSS not to appear, like the blue background when a video was selected

**Review by msadura — APPROVED** [review]

**Pull request merged**